system/Database/Query.php

```diff
@@ -315,17 +315,17 @@ public function getOriginalQuery(): string
 	 *
 	 * @return void
 	 *
-	 * @see https://regex101.com/r/jgAWNQ/2
+	 * @see https://regex101.com/r/EUEhay/1 Test
```

Comment:
you can use "Update Regex" instead of "Save Regex" in regex101.com so the uri path will continue /2 /3 etc instead of new uri start over from /1

Reply:
@samsonasik didn't know about it